_sass/_page-footer.scss

```diff
@@ -0,0 +1,66 @@
+.page-footer {
+  color: $color-gray59;
+  font-size: 12px;
+  min-height: 100px;
+  width: 100%;
+}
+
+.page-footer a {
+  color: $color-eclipse;
+  font-weight: 400;
+  text-decoration: none;
+}
+
+.page-footer a:hover {
+  color: $color-eclipse;
```

Comment:
I noticed that when looking at it, it wasn't super obvious to me that these are links. what do you think about making this the link color? https://github.com/tc39/tc39.github.io/blob/master/_sass/_variables.scss#L5

Reply:
I'm really glad you brought up color.

I was initially going for a so-called &#8220;muted&#8221; aesthetic for the footer, but I like the idea of making the links that color. As you may already know, to be technically correct here (in the context of web a11y), the link color needs to have a contrast ratio > 4.5 to pass AA. I checked, and if we were to use that color with the existing background color, we wouldn't be passing [axe](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd) because (for some odd reason) it rounds the `#fafafa` to `#f0f0f0`.

Proof: https://webaim.org/resources/contrastchecker/?fcolor=017e78&bcolor=f0f0f0&api